packages/lisk-transactions/src/1_second_signature_transaction.ts

```diff
@@ -200,6 +200,7 @@ export class SecondSignatureTransaction extends BaseTransaction {
 		const updatedSender = {
 			...sender,
 			secondPublicKey: this.asset.signature.publicKey,
+			secondSignature: true,
```

Comment:
It should be a number, with values 0 or 1.

Reply:
Done!